bundler/spec/spec_helper.rb

```diff
@@ -69,8 +69,9 @@ def suppress_output
   config.after do
     # Cleanup side effects from cloning git gems, so that they don't interfere
     # with other specs.
-    helper_path = Dependabot::Bundler::NativeHelpers.versioned_helper_path(PackageManagerHelper.bundler_version)
-    FileUtils.rm_rf File.join(helper_path, ".bundle", "bundler")
-    FileUtils.rm_rf File.join(helper_path, ".bundle", "cache", "bundler", "git")
+    helper_path = Dependabot::Utils::BUMP_TMP_DIR_PATH
+    base_path = File.join(helper_path, ".bundle", "ruby", RbConfig::CONFIG["ruby_version"])
```

Comment:
Is `helper_path` used anywhere else? If not, maybe collapse them?

I actually find the naming slightly confusing, I'd expect `helper_path` to be built from `base_path`, not the other way around... Maybe I misunderstand how it works though?

Reply:
I agree it's confusing, I only extracted `base_path` because of a RuboCop LineLength (my least preferred RuboCop rule 😆).

I'll collapse them and "fix" the offense in other way.

Reply:
Done!